test/tools/ossfuzz/protoToYul.h

```diff
@@ -93,15 +95,20 @@ class ProtoConverter
 	void visit(google::protobuf::RepeatedPtrField<T> const& _repeated_field);
 	void registerFunction(FunctionDefinition const&);
 
-	std::string createHex(std::string const& _hexBytes) const;
-	std::string createAlphaNum(std::string const& _strBytes) const;
+	std::string createHex(std::string const& _hexBytes);
+	std::string createAlphaNum(std::string const& _strBytes);
 	bool isCaseLiteralUnique(Literal const&);
 	enum class NumFunctionReturns
 	{
 		None,
 		Single,
 		Multiple
 	};
+	enum class Prefix
```

Comment:
```suggestion
	enum class HexPrefix
```
Since it relates only to hex.

Reply:
Or you could also use `HexPrefix` from `libdevcore/CommonData.h`